node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeMetricsDbMaintainer.java

```diff
@@ -76,7 +76,7 @@ private void handleResponse(MetricsResponse response,
                                 ApplicationId application) {
         if (exception != null) {
             if (failures.get() < maxWarningsPerInvocation)
-                log.log(Level.WARNING, "Could not update metrics for " + application + ": " +
+                log.log(Level.INFO, "Could not update metrics for " + application + ": " +
```

Comment:
What is maxWarningPerInvocation ... should it be reduced.

Reply:
Hm, yes, good point, I think we could just remove it when lowering log level

Reply:
It is set to 2 now, so I am not sure how useful this logging is. But we should probably only do this for prod zones, to avoid noise in zones where we often provision new nodes on deployment